### ISSUE TITLE
Installation instructions on MacOS doesn't work with zsh

### ISSUE THREAD

**Issue by tomekpapiernik** [kyma-project/cli#131]
**Description**

**In iTerm with ZSH** running this command to install Kyma CLI on MacOS: 
```
curl -Lo kyma.tar.gz https://github.com/kyma-project/cli/releases/download/$(curl -s https://api.github.com/repos/kyma-project/cli/releases/latest | grep tag_name | cut -d '"' -f 4)/kyma_Darwin_x86_64.tar.gz \
&& mkdir kyma-release && tar -C kyma-release -zxvf kyma.tar.gz && chmod +x kyma-release/kyma && sudo mv kyma-release/kyma /usr/local/bin \
&& rm -rf kyma-release kyma.tar.gz
```

Fails with this error: 
```
zsh: parse error near `)'
```

When I run the same command in MacOS Terminal with bash, everything works as expected.  

**Expected result**

Can install Kyma CLI using zsh. 

**Actual result**

Can't install with zsh, can install with bash. 

**Steps to reproduce**

Install iTerm, switch shell to zsh and run the command listed in the problem description. 

**Troubleshooting**

I tried Terminal with bash and it worked.

**Comment by piotrmsc** [comment]
I have problems with installing it via homebrew as well :

`brew install kyma-incubator/kyma-incubator/kyma-cli                                                                                                                            

==> Installing kyma-cli from kyma-incubator/kyma-incubator
Warning: A newer Command Line Tools release is available.
Update them from Software Update in System Preferences.

==> Installing dependencies for kyma-incubator/kyma-incubator/kyma-cli: go and dep
==> Installing kyma-incubator/kyma-incubator/kyma-cli dependency: go
==> Downloading https://homebrew.bintray.com/bottles/go-1.12.6.mojave.bottle.tar.gz
==> Downloading from https://akamai.bintray.com/9a/9a60ffb50476baaa34c017b58be8b5a4a7a25b7934b94800ba1f9e6474c117cf?__gda__=exp=1560753304~hmac=312a87d58001c70c76b0841032593bc092adadeee3585f231cc875e4
######################################################################## 100.0%
==> Pouring go-1.12.6.mojave.bottle.tar.gz
🍺  /usr/local/Cellar/go/1.12.6: 9,812 files, 452.7MB
==> Installing kyma-incubator/kyma-incubator/kyma-cli dependency: dep
==> Downloading https://homebrew.bintray.com/bottles/dep-0.5.4.mojave.bottle.tar.gz
==> Downloading from https://akamai.bintray.com/33/33200b5422fac00416ac44c7c28ad5aa627b845cd4d9aeb7002f7d41304deab0?__gda__=exp=1560753327~hmac=3c7d38f868779f4b706ce5ccfa2cc0f0ef90219bafd43228d0a85695
######################################################################## 100.0%
==> Pouring dep-0.5.4.mojave.bottle.tar.gz
🍺  /usr/local/Cellar/dep/0.5.4: 7 files, 11.6MB
==> Installing kyma-incubator/kyma-incubator/kyma-cli
==> Cloning https://github.com/kyma-project/cli.git
Cloning into '/Users/XXX/Library/Caches/Homebrew/kyma-cli--git'...
Note: checking out '49d6bab8036557e6322cf20e31576e7365a93442'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

==> Checking out tag 1.2.0
HEAD is now at 49d6bab Configure goreleaser to release for windows as well (#128)
Error: An exception occurred within a child process:
  RuntimeError: 1.2.0 tag should be db6d31bf0083e50f80d0512f48641344d54346ad
but is actually 49d6bab8036557e6322cf20e31576e7365a93442`

**Comment by a-thaler** [comment]
@piotrmsc The homebrew problem is solved, please re-try. We forgot to adjust the checksum in the homebrew formula

**Comment by suleymanakbas91** [comment]
I see that this is because of a bug in zsh: https://github.com/zsh-users/zsh-autosuggestions/issues/102. A workaround is to disable `bracketed-paste-magic` on zsh.

**Comment by piotrmsc** [comment]
> @piotrmsc The homebrew problem is solved, please re-try. We forgot to adjust the checksum in the homebrew formula

Works 🎉

**Comment by suleymanakbas91** [comment]
Solved by #132
